### DISCUSSION TITLE
Blackjack dealer does not stand on 17

### DISCUSSION THREAD

**Discussion by nbpenn** [deployphp/deployer#3947] [Bugs]
### Deployer Version

v7.3.3

### Target OS

Os X

### Which PHP version are you using?

PHP 8.3

### Content of deploy.php or deploy.yaml

just sayin 

: ) 

![Screenshot 2024-11-04 at 3 37 46 PM](https://github.com/user-attachments/assets/22e6a604-98fb-4a63-a2fa-eabf484fea36)

### Steps to reproduce

1. vendor/bin/dep blackjack
2. Play until dealer gets 17

Expected Behavior:
Dealer stands

Actual Behavior
Dealer hits

**Comment by antonmedv** [comment]
Hmmm.
